Dip/Dip/Dip.swift

```diff
@@ -57,13 +57,35 @@ public final class DependencyContainer {
     configBlock(self)
   }
   
+  // MARK: - Thread safety
+  private func threadSafe(closure: () -> Void) {
+    
```

Comment:
For code style coherence, please remove leading and trailing empty lines inside both those `threadSafe` implementations.